docs/src/intervention.html.jinja2

```diff
@@ -685,6 +751,82 @@
                         </div>
                     </div>
 
+                    <div class="row">
+                        <div class="col-md-5">
+                            <label for="rbadhs">Adenocarcinoma Histology (Predominant)</label>
```

Comment:
This label's `for` attribute should point to rbexs.

Reply:
Quite so. Thanks for the catch; I'll fix that.